### ISSUE TITLE
[A11y] 2.1.1 Keyboard - Unable to Interact with All User Interface Components Using Keyboard Only

### ISSUE THREAD

**Issue by dalonsoa** [research-software-directory/RSD-as-a-service#1764]
See https://github.com/ImperialCollegeLondon/RSD-as-a-service/issues/42 for details
